sig-cluster-lifecycle/charter.md

```diff
@@ -22,9 +22,7 @@ The following topics fall under ownership of this SIG:
 
 #### Cross-cutting and Externally Facing Processes
 
-- This SIG works closely with SIG Release during the end of the cycle, because of how the release of kubeadm is currently managed. The process requires a lock-step coordination which is outlined [here](https://github.com/kubernetes/kubeadm/blob/master/docs/release-cycle.md).
```

Comment:
[2]

Reply:
I think we should may a statement about published release informing jobs.

Reply:
other SIGs are also involved with SIG Release in that regard, but i guess it makes sense adding a note for it.

Reply:
added as [5]